voluncheer/opportunityboard/test_models.py

```diff
@@ -11,9 +14,10 @@
 class OpportunityTest(TestCase):
     """Test cases for Opportunity model"""
 
-    def setUp(self):
+    @classmethod
+    def setUpTestData(cls):
         """See base class."""
-        self.jedi = Organization.objects.create(
+        cls.jedi = Organization.objects.create(
```

Comment:
This should be cls.org instead of clas.jedi

Reply:
Same as above